sw/airborne/firmwares/rotorcraft/autopilot.c

```diff
@@ -573,6 +573,40 @@ bool autopilot_guided_move_ned(float vx, float vy, float vz, float heading)
   return false;
 }
 
```

Comment:
Why do you want to separate the commands here when the message has x,y,z and heading? Are there scenarios you will be updating the horizontal and vertical set-points at different times/frequencies? I guess if you only want to command x,y you can command zeros in z and heading, similar for only setting z.

Also, looking at other functions, vertical velocity is typically referred to at vz so autopilot_guided_set_vz.